### PR TITLE
fix fasta input

### PR DESCRIPTION
**Pull request by d4straub**
Fixed fasta input and activated `test_fasta` that was added in #314 .

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](<https://github.com/>nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/parse_input.nf [hunk 1]
```diff
@@ -38,12 +38,13 @@ def parse_samplesheet(LinkedHashMap row, single_end) {
...
     take:
     input // file.tsv or folder
+    is_fast_input
     single_end
     multiple_sequencing_runs
     extension
 
     main:
-    if ( input.toString().toLowerCase().endsWith(".fasta") || input.toString().toLowerCase().endsWith(".fna") || input.toString().toLowerCase().endsWith(".fa") ) {
+    if ( is_fast_input ) {
```
Comment (erikrikarddaniel) [inline subworkflows/local/parse_input.nf:47]: Should that be `is_fasta_input`? Also above.

File: workflows/ampliseq.nf [hunk 2]
```diff
@@ -63,7 +66,7 @@ max_len = params.max_len ? params.max_len : "Inf"
 
 trunclenf = params.trunclenf ? params.trunclenf : 0
 trunclenr = params.trunclenr ? params.trunclenr : 0
-if ( !single_end && !params.illumina_pe_its && (params.trunclenf == false || params.trunclenr == false) ) {
+if ( !single_end && !params.illumina_pe_its && (params.trunclenf == false || params.trunclenr == false) && !is_fast_input ) {
```
Comment (erikrikarddaniel) [inline workflows/ampliseq.nf:69]: I see you're consistent, so I suppose you have a reason for this name... :-)

**Review by erikrikarddaniel — APPROVED** [review]
Besides the name of the variable, LGTM.

File: subworkflows/local/parse_input.nf (hunk 1 above)
Comment (d4straub) [reply subworkflows/local/parse_input.nf]: Yes, fixed!

**Pull request merged**